api/lib/infrastructure/utils/pdf/attendance-sheet-pdf.js

```diff
@@ -109,30 +129,73 @@ function _drawHeaderLabels({ page, titleFont, pageInformationFont }) {
   });
 }
 
-function _drawCertificationInformationLabels({ page, titleFont, sessionLabelsAndCandidatesInformationFont }) {
-  [
-    [34, 739, 'La session de certification', 9, titleFont],
-    [230, 739, 'N°', 9, titleFont],
-    [34, 715, 'Date :', SESSION_DETAIL_FONT_SIZE, sessionLabelsAndCandidatesInformationFont],
-    [189, 715, 'Heure locale (début) :', SESSION_DETAIL_FONT_SIZE, sessionLabelsAndCandidatesInformationFont],
-    [34, 691, 'Nom du site :', SESSION_DETAIL_FONT_SIZE, sessionLabelsAndCandidatesInformationFont],
-    [189, 691, 'Nom de la salle :', SESSION_DETAIL_FONT_SIZE, sessionLabelsAndCandidatesInformationFont],
-  ].forEach(([x, y, text, fontSize, font]) => {
-    page.drawText(text, {
-      x,
-      y,
-      size: fontSize,
-      font,
-      color: SESSION_DETAIL_DEFAULT_COLOR,
-    });
-  });
+function _drawCertificationInformationLabels({
+  page,
+  titleFont,
+  sessionLabelsAndCandidatesInformationFont,
+  translate,
+  lang,
+}) {
+  const certificationSessionLabel = translate('attendance-sheet.certification-information.session');
+  const numberLabel = translate('attendance-sheet.certification-information.number');
+  const dateLabel = translate('attendance-sheet.certification-information.date');
+  const localTimeLabel = translate('attendance-sheet.certification-information.local-time');
+  const locationNameLabel = translate('attendance-sheet.certification-information.location-name');
+  const roomNameLabel = translate('attendance-sheet.certification-information.room-name');
+
+  const parameters = [
```

Comment:
Je vois que parameters est utilisé à pas mal d'endroits comme nom un peu générique.
Est-il possible de nommer celui-ci plus finement ?
```suggestion
  const labels = [
```
Par exemple ? ou quelque chose de plus pertinent 🙂

Reply:
yes !